api/payment/stripeRouter.js

```diff
@@ -0,0 +1,39 @@
+//IMPLEMENTATION:
+//npm i stripe   ***not "stripe-checkout" but "stripe"***
+//import router into index
+//server router
+//.env file --> "STRIPE_KEY=" --> stripe test account public key
+
+// Code only need uuid installed??( line 15) and to be uncommented
+
+const express = require('express');
+const stripe = require('stripe')(process.env.STRIPE_KEY);
```

Comment:
//in server.js 
if( process.env.Node_ENV  !==  'production') require('dotenv').config();
conSt stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);